modules/libcreate/3.1.0/MODULE.bazel

```diff
@@ -0,0 +1,19 @@
+module(
+    name = "libcreate",
+    version = "3.1.0",
+    compatibility_level = 3,
+)
+
+bazel_dep(name = "googletest", version = "1.14.0")
+bazel_dep(name = "rules_boost", repo_name = "com_github_nelhage_rules_boost")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+
+archive_override(
+    module_name = "rules_boost",
```

Comment:
Does this build without an override? Otherwise it won't be usable by downstream users.

Reply:
Yes, I think we should wait for https://github.com/bazelbuild/bazel-central-registry/pull/1577 /cc @Attempt3035

Reply:
Okay perfect! Asio is the hardest as it has a lot of deps, but I believe it is already in BCR as a standalone, so we can start with using it from there and then move across when it’s done. Will try to work on the others as priority, although no promises on timeline as I’m pretty busy with work at the moment 😝